src/java.base/share/classes/java/lang/Object.java

```diff
@@ -78,11 +78,11 @@ public Object() {}
      *     used in {@code equals} comparisons on the object is modified.
      *     This integer need not remain consistent from one execution of an
      *     application to another execution of the same application.
-     * <li>If two objects are equal according to the {@code equals(Object)}
+     * <li>If two objects are equal according to the {@link equals(Object) equals}
```

Comment:
I believe that should be: `{@link #equals(Object) equals}` (same a few line below).

Reply:
The `{@link equals(Object) equals}` idiom does result in the desired link in the generated javadoc.